notification/src/utils/ctp.js

```diff
@@ -21,6 +23,13 @@ function createCtpClient ({
     fetch
   })
 
+  const userAgentMiddleware = createUserAgentMiddleware({
+    libraryName: packageJson.name,
+    libraryVersion: packageJson.version,
+    contactUrl: packageJson.homepage,
```

Comment:
I am not sure it is good that we add `#readme` to the repo URL as it is then rather a deep link instead of landing/main page.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/391/commits/319e0004c522e428545057877b9b5e4a946e848e